skbeam/core/accumulators/binned_statistic.py

```diff
@@ -72,6 +72,9 @@ def __init__(self, sample, statistic='mean',
             minimum and maximum values along each dimension.
 
         """
+        if mask is None:
+            mask = np.ones_like(sample)
```

Comment:
We need to document the convention of is the mask '1-for-good-pixels' or '1-for-bad-pixels' someplace and make sure we are consistent about it.

attn @CJ-Wright